packages/components/src/Persona.purs

```diff
@@ -102,6 +102,20 @@ pauseSubscription uuid subsno startDate endDate token = do
   where
     authorization = oauthToken token
 
+unpauseSubscription :: UUID -> Int -> Maybe DateTime -> Token -> Aff Subscription
+unpauseSubscription uuid subsno maybeStartDate token = do
+  let maybeStartDateISO = formatDate <$> maybeStartDate
+  callApi usersApi "usersUuidSubscriptionsSubsnoUnpausePost"
+    ([ unsafeToForeign uuid
+     , unsafeToForeign subsno
+     ] <> case maybeStartDateISO of
+            Nothing -> []
+            Just startDateISO -> [ unsafeToForeign { startDate: startDateISO } ]
```

Comment:
Why the `startDate` here though? Didn't we remove it from persona anyway? 🤔

Reply:
Right, I will remove it from here too